ably-rest.gemspec

```
@@ -37,7 +37,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'faraday-typhoeus', '~> 0.2.0'
 
   spec.add_runtime_dependency 'json'
-  spec.add_runtime_dependency 'msgpack', '=1.5.2'
+  spec.add_runtime_dependency 'msgpack', '>= 1.5.2'
```

I don't think we want to allow major versions greater than 1, given that they might introduce a breaking API change and hence no longer function with our SDK. How about:

```suggestion
  spec.add_runtime_dependency 'msgpack', '>= 1.5.2', '< 2'
```
```

---

nice spot, updated

---

👍, looks like some indentation has gone missing now though?

---

oops, fixed